schema/release-schema.json

```
@@ -2283,6 +2293,26 @@
           ]
         }
       }
+    },
+    "Link": {
+      "title": "Link",
+      "description": "A typed connection to a related resource.",
+      "type": "object",
+      "properties": {
+        "rel": {
+          "title": "Link Relation Type",
+          "description": "The semantics of the link, using the open [link relation type](https://standard.open-contracting.org/{{version}}/{{lang}}/schema/codelists/#link-relation-type) codelist.",
```

Based on: "The rel attribute defines the relationship between a linked resource and the current document." https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/rel

```suggestion
          "description": "The relationship to the related resource, using the open [link relation type](https://standard.open-contracting.org/{{version}}/{{lang}}/schema/codelists/#link-relation-type) codelist.",
```

---

Looks good. I used the wording from https://datatracker.ietf.org/doc/html/rfc5988#section-4